script/create_admin.rb

```diff
@@ -48,5 +65,7 @@ def multi_gets(all_text = '')
 puts "\nCopy and paste into the wiki at https://wiki.transformativeworks.org/mediawiki/AO3_Admins:\n"
 
 admins.each do |admin|
-  puts "|-\n| #{Date.today.to_formatted_s("YYYY-MM-dd")} || #{admin.login} || UPDATE WITH USER COMMITTEE"
+  role_description = admin.roles.map { |r| I18n.t("activerecord.attributes.admin/role.#{r}") }.sort.join(", ")
+  role_description = "UPDATE WITH USER COMMITTEE" if role_description.blank?
+  puts "|-\n| #{Time.zone.today.to_formatted_s('YYYY-MM-dd')} || #{admin.login} || #{role_description}"
```

Comment:
For curiosity's sake, whose time zone will this be: the database admin's, the server's (UTC), or the app's (U.S. Eastern)?

Reply:
It's [the app's](https://github.com/otwcode/otwarchive/blob/1c4efeb95b4320832c404255792d504b2f551a01/config/application.rb#L49).

Reply:
Also this line was broken, [`to_formatted_s`](https://api.rubyonrails.org/v5.1/classes/DateTime.html#method-i-to_formatted_s) doesn't expect format strings like that.